client/src/api/auth/getRefreshToken.ts

```diff
@@ -0,0 +1,7 @@
+import api from '../api';
+
+export const getRefreshToken = () => {
+  return api.get('/api/refresh', {
+    withCredentials: true,
+  });
+};
```

Comment:
you dont need new, separated file for each function for request, you can create facade for this purpose